miner/worker.go

```diff
@@ -187,30 +188,34 @@ type worker struct {
 
 	// Needed for randomness
 	db *ethdb.Database
+
+	// Gauge counting the block finalization time(from created to finalized)
+	blockFinalizationTime metrics.Gauge
```

Comment:
How come the metrics is a `gauge` type instead of a `timer` type?

Reply:
FYI, I personally don't really know much about the two, so the question is more for my understanding.